app/build.gradle

```diff
@@ -60,6 +60,8 @@ dependencies {
     compile 'com.github.bumptech.glide:glide:3.7.0'
 
     compile 'it.sephiroth.android.library.bottomnavigation:bottom-navigation:1.0.7'
+
+    compile 'com.android.support:preference-v7:24.2.0'
```

Comment:
We are using Android support libraries such as `com.android.support:appcompat-v7`, `com.android.support:support-v4` and we need to make sure that all these libraries use the same version. Best way to do that is to declare variable for version (see `ext`):
````
ext {
    firebaseVersion = '9.4.0'
    retrofitVersion = '2.1.0'
    supportVersion = '24.2.0'
}
```
And insert variable to maven dependecy. For instance: 
```
    compile "com.android.support:appcompat-v7:$supportVersion"
```